root/etc/cont-init.d/50-config

```diff
@@ -174,6 +174,8 @@ fi
 if [ "$VALIDATION" = "dns" ]; then
   if [ "$DNSPLUGIN" = "route53" ]; then
     PREFCHAL="--dns-${DNSPLUGIN} --manual-public-ip-logging-ok"
+  elif [ "$DNSPLUGIN" = "inwx" ]; then
+    PREFCHAL="-a certbot-dns-${DNSPLUGIN}:dns-${DNSPLUGIN} --certbot-dns-${DNSPLUGIN}:dns-${DNSPLUGIN}-credentials /config/dns-conf/${DNSPLUGIN}.ini --manual-public-ip-logging-ok"
```

Comment:
Why is this needed?

Reply:
I took this from the [documentation](https://github.com/oGGy990/certbot-dns-inwx#usage) of the plugin.
I assumed that this is the way external plugins are called.
Using 
`certbot --dns-inwx`
doesn't work.
I kept the `${DNSPLUGIN}` variables instead of hard coding them to keep the same style as in the lines above.

Reply:
Ahh, so this one's an external one, not an official certbot one. Let me look into it.

Reply:
yup, looks good